flink-connector-pulsar/src/test/java/org/apache/flink/connector/pulsar/testutils/SampleData.java

```diff
@@ -49,7 +49,7 @@ public Foo() {}
 
         @Override
         public String toString() {
-            return "" + i + "," + f + "," + (bar == null ? "null" : bar.toString());
+            return i + "," + f + "," + (bar == null ? "null" : bar.toString());
```

Comment:
I prefer to do only one thing in one PR. This refactor shouldn't be included in the version bump.